tests/constraints/test_cardinality_constraint_continuous.py

```diff
@@ -93,39 +139,30 @@ def test_sampling():
         .all()
     )
 
-    # Assert that we obtain as many (unique!) samples as requested
-    assert len(samples.drop_duplicates()) == N_POINTS
-
-
-# Combinations of cardinalities to be tested
-_cardinalities = sorted(combinations_with_replacement(range(0, 10), 2))
-
 
 @pytest.mark.parametrize(
-    "cardinalities", _cardinalities, ids=[str(x) for x in _cardinalities]
+    "parameter_names", [["Conti_finite1", "Conti_finite2", "Conti_finite3"]]
 )
-def test_random_recommender_with_cardinality_constraint(cardinalities):
+@pytest.mark.parametrize("constraint_names", [["ContiConstraint_5"]])
+@pytest.mark.parametrize("batch_size", [5], ids=["b5"])
+def test_random_recommender_with_cardinality_constraint(
```

Comment:
Why is this test only done with a specific `ContiConstraint` and `batch_size`?

Reply:
My thought for this setup is as follows: 
- The sampling strategy for continuous cardinality constraint has been tested at various cardinality bounds in `test_sampling_cardinality_constraint`. To the best of my understanding, the random recommender is very close to the sampling strategy. To avoid redundancy, I limit the test of random recommender to this specific parameter setting. 
- In the upcoming PR, which enables botorch recommenders for the cardinality constraints and should be ready for review shortly, this test `test_random_recommender_with_cardinality_constraint` will be combined with the test for the botorch recommender. This gives us a single test that tests a meta recommender and follows the same pattern as all the other tests in `test_constraints_continuous.py`. 
Should we expand the test parameters?

Reply:
Ok, if this test will be made redundant soon then I am fine with this.